api/accounts.go

```diff
@@ -80,6 +81,14 @@ func (a *API) enableAccountHandlers() error {
 	); err != nil {
 		return err
 	}
+	if err := a.endpoint.RegisterMethod(
+		"/accounts/{organizationID}/token/transfers/page/{page}",
```

Comment:
LGTM but I'd avoid the `/token/` word.

Reply:
oki i've added it because transfers can lead someone to think "transfer of what?" but since the only thing we can transfer are balances probably is fine to get rid of `token`